src/qt/rpcconsole.cpp

```diff
@@ -461,15 +461,30 @@ void RPCConsole::clear()
                     platformStyle->SingleColorImage(ICON_MAPPING[i].source).scaled(ICON_SIZE, Qt::IgnoreAspectRatio, Qt::SmoothTransformation));
     }
 
+    QFont fontMatch("Monospace");
+    fontMatch.setStyleHint(QFont::TypeWriter);
+    QFontInfo fi(fontMatch);
+    assert(fi.fixedPitch());
+
+    { // DEBUG
+     std::cout << std::endl;
+     std::cout << "family: " << fi.family().toStdString() << std::endl;
+     std::cout << "pixelS: " << fi.pixelSize() << std::endl;
+     std::cout << "pointS: " << fi.pointSize() << std::endl;
+     std::cout << "pointS: (QFont()) " << QFont().pointSize() << std::endl;
+    }
+
     // Set default style sheet
     ui->messagesWidget->document()->setDefaultStyleSheet(
+        QString(
                 "table { }"
                 "td.time { color: #808080; padding-top: 3px; } "
+                "td.message { font-family: %1; font-size: %2; } "
                 "td.cmd-request { color: #006060; } "
                 "td.cmd-error { color: red; } "
                 "b { color: #006060; } "
-                );
-
+            ).arg(fi.family(), "10pt")
```

Comment:
@jonasschnelli Agree on your comment. Would you mind shooting up some new builds for this PR? =)